### PR TITLE
add index file

### PR DESCRIPTION
**Pull request by WhoopDaBase**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+<head>
+<link rel="stylesheet" type="text/css" href="mystyle.css">
+</head>
+<body>
+
+</body>
+</html>
```
Comment (Demango) [inline index.html]: Should have a newline at the end of file

File: index.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+<head>
+<link rel="stylesheet" type="text/css" href="mystyle.css">
```
Comment (Demango) [inline index.html]: Do we have this stylesheet?

**Comment by WhoopDaBase** [comment]
We don't

**Comment by Demango** [comment]
TST

On Sun, Jul 26, 2015 at 4:28 PM, WhoopDaBase <notifications@github.com>
wrote:

> We don't
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Demango/JS-Osu-/pull/1#issuecomment-124986526>.
>



-- 
Edgars Polis

**Comment by Demango** [comment]
@WhoopDaBase 
> We don't

No need to include a reference to a not-yet-existing file

**Pull request closed without merging**